app/controllers/ride_mechanics_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RideMechanicsController < ApplicationController
```

Comment:
Heck yeah! Great controller organization! Well done building this RESTful route to create a joins record